CMakeLists.txt

```diff
@@ -231,18 +229,11 @@ if (NANOGUI_BUILD_GLFW)
   endif()
 
   add_subdirectory(ext/glfw)
-
-  # Two targets have now been defined: `glfw_objects`, which will be merged into
-  # NanoGUI at the end, and `glfw`.  The `glfw` target is the library itself
-  # (e.g., libglfw.so), but can be skipped as we do not need to link against it
-  # (because we merge `glfw_objects` into NanoGUI).  Skipping is required for
-  # XCode, but preferable for all build systems (reduces build artifacts).
-  set_target_properties(glfw PROPERTIES EXCLUDE_FROM_ALL 1 EXCLUDE_FROM_DEFAULT_BUILD 1)
+  list(APPEND NANOGUI_LIBS glfw)
 
   mark_as_advanced(
-    GLFW_BUILD_DOCS GLFW_BUILD_EXAMPLES GLFW_BUILD_INSTALL GLFW_BUILD_TESTS
-    GLFW_DOCUMENT_INTERNALS GLFW_INSTALL GLFW_USE_CHDIR GLFW_USE_MENUBAR
-    GLFW_USE_OSMESA GLFW_VULKAN_STATIC GLFW_USE_RETINA GLFW_USE_MIR
+    GLFW_BUILD_DOCS GLFW_BUILD_EXAMPLES GLFW_BUILD_TESTS
+    GLFW_INSTALL GLFW_USE_OSMESA GLFW_VULKAN_STATIC
```

Comment:
Removed options that GLFW no longer uses.

Reply:
Been looong time since I looked under hood at NanoGUI. Why did you drop GLFW_USE_MENUBAR? I'm asking because I wonder if this precludes having a nice menu bar with NanoGUI on Wayland. If I'm off in the weeds with this question feel free to say so. I've been deep in CopperSpice and CsScintilla lately.

Reply:
It was [removed ](https://github.com/glfw/glfw/blob/fe2ac5027b617234670d400f800c4b6f1e5f24bb/docs/news.dox#L342)in GLFW 3.3.

Reply:
Thank you for the link!